src/utils/sources-tree/getURL.js

```diff
@@ -45,65 +47,75 @@ export function getURL(sourceUrl: string, debuggeeUrl: string = ""): ParsedURL {
 
     case "moz-extension:":
     case "resource:":
-      return merge(def, {
+      return {
+        ...def,
         path,
-        group: `${protocol}//${host || ""}`,
-        filename
-      });
+        filename,
+        group: `${protocol}//${host || ""}`
+      };
 
     case "webpack:":
     case "ng:":
-      return merge(def, {
+      return {
+        ...def,
         path: path,
-        group: `${protocol}//`,
-        filename: filename
-      });
+        filename,
+        group: `${protocol}//`
+      };
 
     case "about:":
       // An about page is a special case
-      return merge(def, {
+      return {
+        ...def,
         path: "/",
-        group: url,
-        filename: filename
-      });
+        filename,
+        group: url
+      };
 
     case "data:":
-      return merge(def, {
+      return {
+        ...def,
         path: "/",
         group: NoDomain,
         filename: url
-      });
+      };
 
     case null:
       if (pathname && pathname.startsWith("/")) {
         // use file protocol for a URL like "/foo/bar.js"
-        return merge(def, {
+        return {
+          ...def,
           path: path,
-          group: "file://",
-          filename: filename
-        });
-      } else if (host === null) {
```

Comment:
Finally figured it out; removing this `else if` makes `browser_dbg-sources-named-eval.js` fail because it doesn't start with "/" and we have no other handling here, so the source essentially gets "lost".

When I make this change:

```diff
diff --git a/src/utils/sources-tree/getURL.js b/src/utils/sources-tree/getURL.js
index 8c0687dee..d3c354f18 100644
--- a/src/utils/sources-tree/getURL.js
+++ b/src/utils/sources-tree/getURL.js
@@ -90,6 +90,14 @@ function _getURL(source: Source): ParsedURL {
           group: "file://"
         };
       }
+
+      return {
+        ...def,
+        path: path,
+        filename,
+        group: "example.com"
+      };
+
```

The test passes because I've hardcoded the domain.  I think we need to use the `defaultDomain` var that was removed from this first commit.